src/KubernetesConfiguration/docs/Get-AzKubernetesExtension.md

```diff
@@ -1,7 +1,7 @@
 ---
 external help file:
 Module Name: Az.KubernetesConfiguration
-online version: https://docs.microsoft.com/powershell/module/az.kubernetesconfiguration/get-azextension
+online version: https://docs.microsoft.com/powershell/module/az.kubernetesconfiguration/get-azkubernetesextension
```

Comment:
@wyunchi-ms  FYI I have to fix a few online version urls otherwise CI will fail.
Not sure why this was not checked in the original kube configuration PR.